.changeset/new-dragons-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ag.ds-next/docs': minor
+---
+
+docs: Update Storybook to version 8
```

Comment:
Storybook isn't technically part of docs, but I think it's useful info. The next and the playroom updates are relevant though, so I would also add them

Reply:
- [ ] add next and playroom to changeset

Reply:
Need to update to include react package changes as well (tests etc)

Reply:
Tests don't affect NPM packages but the deps changes may. I'll update this as well.